README.md

```diff
@@ -1,2 +1,185 @@
-# Budget_App
-Budget-App is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget App ](#-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage ](#usage-)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👤 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-the-project"></a>
+The **Budget App** mobile web application where you can manage your budget: you have a list of expenses associated with a group, so that you can see how much money you spent and on what. The app allows users to:
+
+- *Register and log in, so that the data is private to them.*
+- *Introduce new expenses associated with a group.*
+- *See the money spent on each group.*
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul> 
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/ror">Linters</a></li>
+
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">ROR</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Groups Page:** *Displays a list of groups with their name, icon and total amount of expenses*
+- **Expenses page:** *Display a list of expenses for a given group, ordered by the most recent.*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Presentation video 🔗](http://test.nogdevhouse.com/presentation.mp4)
+- [app link](https://budget-app-ggml.onrender.com/)
```

Comment:
- [ ] It will be great to ensure that the user receives the confirmation instructions. Since you have enabled this feature in your application, it is impossible to test your live application since it is requesting the user to confirm his email address. Kindly remember that the reviewer has to test both the category page and the transactions page in your live application also. Kindly fix this issue